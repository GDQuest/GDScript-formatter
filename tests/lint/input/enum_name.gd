#region Good

enum GoodEnum {}

#endregion

#region Bad

enum badEnum {}
enum bad_enum {}
enum BAD_ENUM {}
enum Bad_Enum {}
enum _bad_enum {}

#endregion