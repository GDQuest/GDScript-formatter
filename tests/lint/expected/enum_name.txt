tests/lint/input/enum_name.gd:9:enum-name:error: Enum name 'badEnum' should be in PascalCase format
tests/lint/input/enum_name.gd:10:enum-name:error: Enum name 'bad_enum' should be in PascalCase format
tests/lint/input/enum_name.gd:11:enum-name:error: Enum name 'BAD_ENUM' should be in PascalCase format
tests/lint/input/enum_name.gd:12:enum-name:error: Enum name 'Bad_Enum' should be in PascalCase format
tests/lint/input/enum_name.gd:13:enum-name:error: Enum name '_bad_enum' should be in PascalCase format